Makefile

```diff
@@ -1,17 +1,27 @@
 TEST_TARGETS=zhenech_DASH_katello-client\:el5 zhenech_DASH_katello-client\:el6 zhenech_DASH_katello-client\:el7
 USE_SELINUX=$(shell test -d /sys/fs/selinux && echo ":Z")
 
+test-python3:
```

Comment:
this target is currently not executed on travis, intentional?

Reply:
I don't think it would work since the tests require dnf, tracer, etc